models/things.js

```diff
@@ -145,4 +145,8 @@ var Things = {
    }
  };
 
+AdapterManager.on('thing-added', function(thing) {
+  Things.handleNewThing(thing);
+});
```

Comment:
Something I want to note here because I just confirmed it for myself: the emitted `thing-added` event is sending over the correct `device.asThing()` format, not a raw Thing object.